ipyevents/events.py

```diff
@@ -13,6 +13,17 @@ class Event(CoreWidget):
     watched_events = List().tag(sync=True)
     ignore_modifier_key_events = Bool(False).tag(sync=True)
     prevent_default_action = Bool(False).tag(sync=True)
+    xy_trait_coords = Unicode(allow_none=True, default=None).tag(sync=True)
+    _xy_trait_coords_allowed = List([
+        None,       # Not tracking mouse x/y
+        'array',    # "natural" coordinates for the widget (e.g. image)
+        'client',   # Relative to the visible part of the web page
+        'offset',   # Relative to the padding edge of widget
+        'page',     # Relative to the whole document
+        'relative', # Relative to the widget
+        'screen'    # Relative to the screen
+    ]).tag(sync=True)
```

Comment:
Does `_xy_trait_coords_allowed` need to be serialized? Unless I'm missing something, I'm not sure you're using it in the client.